### PR TITLE
Remove tests mock

### PR DESCRIPTION
**Pull request by maage**
Package has inside it old version v1.0.1 of unittest mock source module. Even if it is just used to testing, I think it is better to use packaged modules instead.

### CODE REVIEW

File: tests/support.py [hunk 1]
```diff
@@ -31,7 +31,9 @@
 if PY3:
     from unittest import mock
 else:
-    from . import mock
+    import backports.unittest_mock
+    backports.unittest_mock.install()
+    from unittest import mock
```
[the remark this answers is not in the document]
Comment (maage) [reply tests/support.py]: I've added patch to silence this warning.

**Comment by maage** [comment]
I merged flake8 patch to make it more simple still.

**Comment by lukash** [comment]
Hello, sorry about being so late. I've actually created a PR for dropping Python 2 altogether: https://github.com/rpm-software-management/dnf-plugins-extras/pull/179

**Comment by lukash** [comment]
https://github.com/rpm-software-management/dnf-plugins-extras/pull/179 was merged, closing.

**Pull request closed without merging**